scripts/get_assets

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+#    Copyright 2013-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+This module downloads external dependencies of assets used by Workload Automation.
+Works by first downloading a directory index of the assets, then iterating through
+it to get assets for the specified workloads.
+"""
+import sys
+import argparse
+import logging
+
+from wlauto import Workload
+from wlauto.common.resources import File
+from wlauto.resource_getters.standard import HttpGetter
+from wlauto.utils.log import init_logging
+
+__version__ = '0.1.0'
+
+# Get index.json from this URL
+ASSETS_REPO = 'https://github.com/jimboatarm/wa-assets/raw/master/workloads'
```

Comment:
Agreed, it will be configurable.
It's only like this because at the moment we need a way to change/test the folder structure without asking for commit access to ARM-software.